{{cookiecutter.repo_name}}/src/{{cookiecutter.package_name}}/{{cookiecutter.c_extension_module}}.c

```diff
@@ -28,11 +28,15 @@ static PyObject* {{ cookiecutter.c_extension_function }}(PyObject *self, PyObjec
     PyObject *kwargs;
     PyObject *result;
 
+{%- if cookiecutter.legacy_python == "yes" %}
     #if PY_MAJOR_VERSION < 3
       module = PyImport_ImportModule("__builtin__");
     #else
+{%- endif %}
```

Comment:
We probably need to have some creative positioning of the endif tag to make the indentation appear "natural". The goal of the template is to produce well looking output.

Reply:
I was trying. I may need to reevaluate the templates again.

Reply:
Try this:
```
    #else
  {% endif %}    module = PyImport_ImportModule("builtins");
```

Reply:
I'm almost there.
I found two variants of macro indentation...
- https://github.com/ionelmc/cookiecutter-pylibrary/blob/052b10d6b5ac4067b685b9e202b6ba330023ddcc/%7B%7Bcookiecutter.repo_name%7D%7D/src/%7B%7Bcookiecutter.package_name%7D%7D/%7B%7Bcookiecutter.c_extension_module%7D%7D.c#L31-L35
- https://github.com/ionelmc/cookiecutter-pylibrary/blob/052b10d6b5ac4067b685b9e202b6ba330023ddcc/%7B%7Bcookiecutter.repo_name%7D%7D/src/%7B%7Bcookiecutter.package_name%7D%7D/%7B%7Bcookiecutter.c_extension_module%7D%7D.c#L98-L102
Can I simplify it to the second one that is flush to the left? or should I indent the second one, so it is better foldable (in editors) inside the function?

Reply:
Ooof you're right. Lets keep it simple and not ident macros at all.

Reply:
Ok